docs/tutorial_custom.rst

```diff
@@ -100,12 +100,12 @@ to automatically apply the method upon ever load.
 
 .. code:: python
```

Comment:
Also need an example of attaching multiple custom functions upon Instrument initialization.